docs/source/api_ref_utilities.rst

```diff
@@ -4,6 +4,19 @@ torchtune.utils
 
 .. currentmodule:: torchtune.utils
 
+
+.. _checkpointing_label:
+
+Checkpointing
+-------------
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    FullModelHFCheckpointer
+    FullModelMetaCheckpointer
```

Comment:
Wait I thought we were doing away with `FullModel`?

Reply:
Oh sorry, decided to punt the change to a separate PR since that will be a fairly widespread change.